README.md

```diff
@@ -112,6 +112,7 @@ Due to an issue with ESLint, config extension cannot be called from a globally i
 * [Keith Cirkel](https://github.com/keithamus) for painstakingly formatting all of ESLint's rules into JSON in [eslint-config-strict](https://github.com/keithamus/eslint-config-strict)
 * [AirBnB](https://github.com/airbnb/javascript) for sharing all of their config in [JavaScript Style Guide](https://github.com/airbnb/javascript)
 * [Google](https://google.github.io/styleguide/javascriptguide.xml) for sharing their styleguide
+* [ES-Next Compat Table](https://github.com/kangax/compat-table) for the excellent docs on node features
```

Comment:
:+1:  Nice reference!

Maybe also link to https://kangax.github.io/compat-table/es6/#node4 ?